src/test/kotlin/TestServer.kt

```diff
@@ -297,10 +297,10 @@ object TestServer {
 
         class A(val str: String) : Base()
 
-        class B(val i: @Min(0) @Max(2) Int) : Base()
+        class B(@Min(0) @Max(2) val i: Int) : Base()
 
         @WithExample
-        class C(val l: @Clamp(0, 10) Long) : Base() {
+        class C( @Clamp(0, 10) val l: Long) : Base() {
```

Comment:
Were these annotations used on purpose between the name and the type of the constructor parameter? I modified this, because it caused problems as using this way the annotation instance didn't have an errorMessage property, only value property.

Reply:
Yes, at the beginning the annotations were only supported on types due to some issue, i believe it is fixed now and didn't really change that to the more natural syntax.

Reply:
Ok, it was new for me. I didn't see any annotations on types before.
It seems like the annotation parameters with default values are ignored if I use the annotation on the type.

It causes this error on start:
```
kotlin.reflect.jvm.internal.KotlinReflectionInternalError: Method is not supported: public abstract java.lang.String com.papsign.ktor.openapigen.annotations.type.string.length.Length.errorMessage() (args: [])
```
In the `getConstraint` method of the `LengthProcessor` class I can see with the debugger, that there are values for min and max only, the errorMessage is missing.